API/src/Modules/Reports/Momentum.Reports.Infrastructure/Repositories/ReportRepository.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Marten;
+using Momentum.Framework.Infrastructure.Repositories;
+using Momentum.Reports.Core.Models;
+using Momentum.Reports.Core.Repositories;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Momentum.Reports.Infrastructure.Repositories
+{
+    public class ReportRepository : GenericTimeTrackedRepository<Report>, IReportRepository
+    {
+        private readonly IDocumentStore _store;
+
+        public ReportRepository(IDocumentStore store) : base(store)
+        {
+            _store = store;
+        }
+
```

Comment:
`GenericTimeTrackedRepository` provides this property: `.Store`, remove all this - and change `_store` usages to `Store`